### ISSUE TITLE
CTD when cycling through ECAM Systems - Experimental 24.9.2021

### ISSUE THREAD

**Issue by PM3Delta** [flybywiresim/aircraft#5904]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2021-09-24T15:56:41+00:00",
  "ref": "experimental",
  "sha": "f96ac408fa31686be361dc8c2907a5de78b011f8",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

Every now and then I get a CTD when cycling through the ECAM System displays with the ALL button.

My system is otherwise very stable.  9700K, 2070S, 32GB, NVMe, at 2K and I never really have CTD.

Could you please look into it, if you get the time?  Thx.

### Expected behavior

No CTD.

### Steps to reproduce

Cycle through the ECAM systems pages using the ALL button.

### References (optional)

![image](https://user-images.githubusercontent.com/60411051/134765142-2ac87f3e-2d3a-41ac-876a-549ab6d7de6a.png)

![image](https://user-images.githubusercontent.com/60411051/134765147-79456869-b312-4fbf-b499-197b99614c46.png)

**Comment by igormagun** [comment]
I've gotten a similar crash a few times now on the latest development version - though instead of cycling through with the "All" button, I would go through the different page buttons manually one-by-one (I always do this as part of my pre-flight flows), and typically crash when opening the status page (STS button). 

I've ruled out most of my mods at this point - only other mod I had running at the time was the AIG live traffic workaround. Seems unlikely that it would cause such an issue but I can't rule it out, as I've struggled to consistently reproduce this bug.
